### PR TITLE
Support creating Chronos from ChronosTime

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

File: tests/TestCase/DateTime/ConstructTest.php [hunk 1]
```diff
@@ -151,6 +152,24 @@ public function testCreateFromChronosDate()
         $this->assertSame('2021-01-01 00:00:00', $chronos->format('Y-m-d H:i:s'));
     }
 
+    public function testCreateFromChronosTime()
+    {
+        $time = new ChronosTime('20:14:12.123456');
+        $chronos = new Chronos($time);
+        $this->assertSame('20:14:12.123456', $chronos->format('H:i:s.u'));
```
Comment (markstory) [inline tests/TestCase/DateTime/ConstructTest.php]: Should we also have assertions for the date component of `Chronos` to help prevent regressions?

Comment (othercorey) [reply tests/TestCase/DateTime/ConstructTest.php]: It will always be the current date. I could check that?

Comment (othercorey) [reply tests/TestCase/DateTime/ConstructTest.php]: Should we special case using the test now date with a ChronosTime time or should it always be the current date?

Comment (markstory) [reply tests/TestCase/DateTime/ConstructTest.php]: I think having a test that it is the current date would be good for future us.

Comment (othercorey) [reply tests/TestCase/DateTime/ConstructTest.php]: Extended tests to compare full Chronos string.

**Comment by markstory** [comment]
Looks good. I don't know what is up with psalm.

**Pull request merged**